policyengine_us/tests/policy/baseline/gov/states/de/tax/income/subtractions/de_itemized_deductions.yaml

```
@@ -0,0 +1,57 @@
+- name: filling status = SEPERATE, total salt < salt cap
```

and other test names
```suggestion
- name: Separate filing status, total SALT < SALT cap
```